xchange-acx/src/main/java/org/knowm/xchange/acx/AcxMapper.java

```diff
@@ -133,4 +157,28 @@ public String getOrderType(OrderType type) {
     }
     throw new IllegalArgumentException("Unknown order type: " + type);
   }
+
+  public UserTrade mapTrade(AcxTrade trade) {
+    return new UserTrade.Builder()
+        .currencyPair(mapCurrencyPair(trade.market))
+        .id(trade.id)
+        .orderId(trade.orderId)
+        .price(trade.price)
+        .originalAmount(trade.volume)
+        .timestamp(trade.createdAt)
+        .type(mapOrderSide(trade.side))
+        .build();
+  }
+
+  private Map<String, CurrencyPair> marketMap = new ConcurrentHashMap<>();
```

Comment:
move this declaration to the beginning of he class

Reply:
done